### DISCUSSION TITLE
Potential discussion topics for LinkedMusic technical session

### DISCUSSION THREAD

**Discussion by dchiller** [DDMAL/linkedmusic-datalake#21] [General]
Let's collect ideas/questions/starting topics for our technical discussion of LinkedMusic on 20 October.

**Comment by malajvan** [comment]
MusicBrainz specific discussions with Alastair (?):
1. MusicBrainz exports: 
   - Currently requiring local instance of MusicBrainz Server for postgres dump? 
   - Should we use the PostgresQL dump or the JSON dump?
2. Linked Music overriding the schema.org context with Wikidata mappings
    see #23 
    - Are we doing it right?
    -  How to handle non-existing/weird mappings:
        -  Ex: releaseDate doesn't have equivalent on Wikidata. Wikidata has P577 (publication date) that represents this notion of releaseDate but links to schema.org publicationDate rather than releaseDate. -> should we keep releaseDate or use publicationDate??
        - Ex: thumbnail only exists as Qs in wikidata
        - Ex: recordLabel on schema.org doesn't link to record label (P264) on wikidata
